src/components/AccountsPage/AccountGridItem/Header.js

```diff
@@ -56,10 +56,7 @@ class Header extends PureComponent<{
     return (
       <Box flow={4}>
         <Box horizontal ff="Open Sans|SemiBold" flow={3} alignItems="center">
-          <ParentCryptoCurrencyIcon
-            currency={currency}
-            parentCurrency={parentAccount && parentAccount.currency}
-          />
+          <ParentCryptoCurrencyIcon currency={currency} parent={parentAccount} />
```

Comment:
parent does not exist as a prop in ParentCryptoCurrencyIcon 🤔

Reply:
nevermind, i didn't see the related change.

are you sure there are no other reference of that component?

Reply:
I am